src/vm/exceptionhandling.cpp

```diff
@@ -5186,10 +5186,49 @@ BOOL IsSafeToHandleHardwareException(PCONTEXT contextRecord, PEXCEPTION_RECORD e
         IsIPInMarkedJitHelper(controlPc));
 }
 
+#ifdef _TARGET_ARM_
+static inline BOOL
+HandleArmSingleStep(PCONTEXT pContext, PEXCEPTION_RECORD pExceptionRecord,
+                    Thread *pThread)
+{
+#ifdef LINUX32
```

Comment:
This condition is not needed. The code is compiled for Unix only and you already have `_TARGET_ARM_` ifdef around it.